### PR TITLE
Add changeName functionality 

### PR DESCRIPTION
**Pull request by b-pl**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,17 +10,34 @@
...
+                <p id="welcomeMsg"></p>
 
                 <button class="gameMenuButton" id="newGameButton"
                     value="New Game">New Game</button>
                 <button class="gameMenuButton" id="highScoresButton"
                     value="High-Scores">High-Scores</button>
-                <button class="gameMenuButton" id="changePleyerButton"
-                    value="Change Player">Change Player</button>
+                <button type="button" class="gameMenuButton" id="changePlayerButton"
+                    value="Change Player" onClick="modal.style.display = 'block'">Change Player</button>
```
Comment (iksemot) [inline index.html]: Move content of `onClick` to a separate function in script.js. Having a clear separation between entities is a general good practice (in this case between HTML template and the JS logic)

File: scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// let template = document.getElementById('testTemplate');
+// let tempContent = template.content;
+// let oldChild = document.getElementById('gameMenu');
+// // let newChild = document.getElementById('testTemplate');
+// gameWindow.replaceChild(tempContent, oldChild);
+
+var playerName = "Adolf";
+
+// Display welcome msg
+document.getElementById('welcomeMsg').innerHTML = "Welcome, " + playerName + "!";
```
Comment (iksemot) [inline scripts/script.js]: Common practice nowadays is to use querySelector (https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector)

File: index.html (hunk 1 above)
Comment (b-pl) [reply index.html]: Done

File: scripts/script.js (hunk 2 above)
Comment (b-pl) [reply scripts/script.js]: Done

**Pull request merged**